integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -453,7 +437,7 @@ def _query_result_to_documents(result: QueryResult) -> List[List[Document]]:
             for j in range(len(answers)):
                 document_dict: Dict[str, Any] = {
                     "id": result["ids"][i][j],
-                    "content": documents[i][j],
+                    "content": answers[j],
```

Comment:
Why was this change made? Doesn't seem related.

Reply:
I think it was modified while I was trying to use --fix from lint scripts. Will fix it.

Reply:
Without this change it raises a linting error, which is also kinda valid.